packages/read-model/tests/Unit/Image/ImageViewRepositoryTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ReadModelBundle\Unit\Image;
+
+use PHPUnit\Framework\TestCase;
+use Shopsys\FrameworkBundle\Component\Image\Image;
+use Shopsys\FrameworkBundle\Component\Image\ImageFacade;
+use Shopsys\ReadModelBundle\Image\ImageView;
+use Shopsys\ReadModelBundle\Image\ImageViewFactory;
+use Shopsys\ReadModelBundle\Image\ImageViewRepository;
+
+class ImageViewRepositoryTest extends TestCase
+{
+    private const IMAGE_EXTENSION = 'jpg';
+
+    /** @var \Shopsys\FrameworkBundle\Component\Image\ImageFacade */
+    private $imageFacadeMock;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->imageFacadeMock = $this->createMock(ImageFacade::class);
+        $this->imageFacadeMock->method('getImagesOrNullsByEntitiesIndexedByEntityId')
+            ->willReturnCallback(function ($entityIds, string $entityClass) {
+                $images = [];
+
+                foreach ($entityIds as $entityId) {
+                    // id 800 represents not existing product
+                    if ($entityId === 800) {
```

Comment:
```suggestion
                    if ($entityId === self::NOT_EXISTING_PRODUCT_ID) {
```
I suggest to name the ID and use it everywhere in the test. You can remove the comment then